services-camel-app/src/test/java/edu/unc/lib/boxc/services/camel/enhancements/EnhancementRouterIT.java

```diff
@@ -149,6 +174,10 @@ public void init() throws Exception {
         FileUtils.writeStringToFile(audioScriptFile, "exit 0", "utf-8");
         audioScriptFile.deleteOnExit();
 
+        File videoScriptFile = new File("target/convertVideo.sh");
```

Comment:
Is this needed anymore? Mp44u is used instead of the shell scripts now, so maybe the audio script a couple lines up can go too?